src/handlers/net/NetStreamHandler.h

```diff
@@ -78,8 +82,11 @@ class NetworkMetricsBucket final : public visor::AbstractMetricsBucket
         , _topASN("packets", "asn", {"top_ASN"}, "Top ASNs by IP")
         , _topIPv4("packets", "ipv4", {"top_ipv4"}, "Top IPv4 IP addresses")
         , _topIPv6("packets", "ipv6", {"top_ipv6"}, "Top IPv6 IP addresses")
+        , _size("packets", {"size"}, "Quantiles of packets size")
```

Comment:
please make this `payload_size` ... that way it it's a bit more specific than just "size" but fits for both packet and e.g. dnstap. and please match the variable name. in this description, change to "Quantiles of payload sizes, in bytes"